01_fib/choohy/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+
+	fmt.Fprintln(out, "Call Fibonacci numbers!")
+	fib(7)
+}
+
+func fib(n int) {
+	i := 0
+	previous := 0
+	current := 1
+	next := 1
```

Comment:
I don't understand.

Reply:
`ineffectual` ~ has no effect.
you can declare next only inside the loop body:
```go
next := current + previous
```
and it will have the same effect.